Program.cs

```diff
@@ -1,21 +1,52 @@
 ﻿using MiniGame.App;
-using MiniGame.App.Warriors;
 using System.Text;
+using MiniGame.App.HW2;
 
 Console.OutputEncoding = Encoding.Unicode;
 
-var warrior1 = Menu.Hero(1);
-var warrior2 = Menu.Hero(2);
+do
+{
+    List<Warrior> warriors = new List<Warrior>();
 
-int addHero1 = SuperPowerGenerator.GetRandom();
-Console.WriteLine("\n Що ви хочете покращити першому Герою №1 на [" + addHero1 + "] пунктів");
-Menu.Update(warrior1, addHero1);
+    int iter = 1;
```

Comment:
why do you need that field?

Reply:
Just to show number of hero in Console (like in the next remark, "Hero №{iter}")

Console.WriteLine($"\n What do you want to increase Hero №{iter--} by [{addToHero}] points");